packages/cli/src/commands/login.ts

```diff
@@ -0,0 +1,21 @@
+import { RedoclyClient } from '@redocly/openapi-core';
+import { blue, green } from 'colorette';
+import { promptUser } from '../utils';
+import { getRawConfigContent } from '../../../core/src/config/load';
+import { Region } from '@redocly/openapi-core';
```

Comment:
join this with first import { RedoclyClient, Region } from '@redocly/openapi-core';